sentry/src/main/java/io/sentry/transport/StdoutTransport.java

```diff
@@ -15,14 +15,13 @@ public StdoutTransport(final @NotNull ISerializer serializer) {
   }
 
   @Override
-  public TransportResult send(final @NotNull SentryEnvelope envelope) throws IOException {
+  public void send(final @NotNull SentryEnvelope envelope, Object hint) throws IOException {
     Objects.requireNonNull(envelope, "SentryEnvelope is required");
 
     try {
       serializer.serialize(envelope, System.out);
-      return TransportResult.success();
     } catch (Exception e) {
-      return TransportResult.error();
+      // do nothing
```

Comment:
not really against but why the change?

Reply:
The interface has changed and it does not return anymore.